listenbrainz/utils.py

```diff
@@ -118,3 +118,25 @@ def connect_to_rabbitmq(username, password,
             error_message = "Cannot connect to RabbitMQ: {error}, retrying in {delay} seconds."
             error_logger(error_message.format(error=str(err), delay=error_retry_delay))
             time.sleep(error_retry_delay)
+
+
+def init_cache():
+    """ Initializes brainzutils cache. """
+    from brainzutils import cache
+    from listenbrainz import default_config as config
```

Comment:
These imports are a bit ugly (and I plan to remove all such imports soon, using flask app config values), I'd prefer it if init_cache would take args host, port etc. That would also mean we don't need to import logging here.